### DISCUSSION TITLE
Update to Italian locale [it]

### DISCUSSION THREAD

**Discussion by Furio92** [aclap-dev/video-downloadhelper#786] [Language & Translation]
Hi there!

I translated every non-translated string (for v9.0) and updated the translation of many other strings that were (IMO) poorly translated.
I couldn't find a way to export the translation file into a JSON, therefore I'm uploading it as a .txt.

[it.txt](https://github.com/aclap-dev/video-downloadhelper/files/15501557/it.txt)

(And thank you very much for your work!)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
